mifosng-android/src/main/java/com/mifos/api/DataManager.java

```diff
@@ -178,4 +181,18 @@ public DataManager(BaseApiManager baseApiManager,
     public Observable<Page<Charges>> getListOfCharges(int clientId) {
         return mBaseApiManager.getLoanApi().getListOfCharges(clientId);
     }
+
+    /**
+     * CollectionSheet API
+     */
+
+    public Observable<IndividualCollectionSheet> getIndividualCollectionSheet(
```

Comment:
Why are you using DataManager class, @tarun0 Make a class with name `DataManagerCollectionSheet` and use that.